docs/docs/configuration/alpha_config.md

```diff
@@ -309,6 +309,7 @@ Provider holds all configuration for a single provider
 | `loginURL` | _string_ | LoginURL is the authentication endpoint |
 | `redeemURL` | _string_ | RedeemURL is the token redemption endpoint |
 | `profileURL` | _string_ | ProfileURL is the profile access endpoint |
+| `introspectURL` | _string_ | IntrospectURL is the claims and permissions access Endpoint (OIDC) |
```

Comment:
Is this an OIDC specific thing or can it be used across multiple providers? Eg is this a more generic concept that might be available on for example, Azure?

Reply:
Out of interest, are you using Okta? I know with Okta it returns thin tokens for the default authentication server but you can reconfigure it to send fat tokens with more information in if you prefer (I've done this for testing before)

That said, I don't think that precludes us from introducing this :)

Reply:
No @JoelSpeed , we are working with Philips HSDP provider which provides the authorization context via introspection endpoint.